app.js

```diff
@@ -13,6 +13,52 @@ function hasFilter(name, filters) {
   return false;
 }
 
+function searchToUrl(search) {
+  if (!search.toString()) {
+    return "";
+  }
+  return "?" + search.toString();
+}
+
+function initialSetStateByUrl() {
```

Comment:
Ah, sounds fine then, can you rename the function to setDefaultHistoryState() then ? It would be more clear this way.